app/src/main/java/com/xa/xpensauditor/EmailSender.java

```diff
@@ -16,20 +16,36 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+/**
+ * This class is a async task to send emails using the MailJet API. This does
+ * not block the main thread and can be used from the UI. The API is rate-limited to
+ * 500 emails per day.
+ */
 public class EmailSender extends AsyncTask<Void, Integer, Boolean> {
 
     private String email;
     private String amt;
     private String shop;
     private String cat;
 
+    /**
+     * @param email - The email id of the receiver
+     * @param amt - The amount of the transaction
+     * @param shop - The shop where the transaction occurred
+     * @param cat - The category of the transaction
+     */
     public EmailSender(String email, String amt, String shop, String cat) {
         this.email = email;
         this.amt = amt;
         this.shop = shop;
         this.cat = cat;
     }
 
+    /**
+     * sends message to the e-mail ID provided in the constructor.
+     * @param message - The message to be sent.
+     * @return
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.